### PR TITLE
nischay.is-a.dev | Add file nischay.json

### PR DESCRIPTION
**Pull request by nischay02**

### CODE REVIEW

File: domains/nischay.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "description": "OGGY's Main Website",
+  "github": "https://github.com/nischay876",
+  "owner": {
+    "username": "nischay876",
+    "email": "nischay876@gmail.com",
+    "instagram": "https://www.instagram.com/nischay_02/",
+    "snapchat": "https://snapchat.com/add/nischay_02",
```
Comment (MaskDuck) [inline domains/nischay.json:8]:
```suggestion
    "snapchat": "nischay_02",
```

File: domains/nischay.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "description": "OGGY's Main Website",
+  "github": "https://github.com/nischay876",
+  "owner": {
+    "username": "nischay876",
+    "email": "nischay876@gmail.com",
+    "instagram": "https://www.instagram.com/nischay_02/",
```
Comment (MaskDuck) [inline domains/nischay.json:7]:
```suggestion
    "instagram": "nischay_02",
```

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Comment by MaskDuck** [comment]
do you see our comments

**Comment by nischay876** [comment]
> do you see our comments
yes but i don 't understand that is there something that i need to edit or the PR is just on hold from your side?

**Review by TheBotlyNoob — APPROVED** [review]

**Comment by MaskDuck** [comment]
@nischay876 we will remove one of your txt record

**Comment by nischay876** [comment]
ok remove the google verification one bcz it got reset

**Pull request merged**